src/modules/user/dto/create-user-response.dto.ts

```diff
@@ -10,4 +11,7 @@ export class CreateUserResponseDto {
   @IsNotEmpty()
   public password!: string;
 
+  //@isEnum(UserTypeEnum) ---> Why it`s not work
+  public role!: UserTypeEnum;
```

Comment:
як я вже казав тип повинен бути не обовʼязковим і по дефолту кастомер
```suggestion
  @isEnum(UserTypeEnum)
  public role?: UserTypeEnum;
```

ти імпортнув UserTypeEnum як тип, тому не працює